_posts/2019-11-25-z-index-is-confusing.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: post
+title: "z-index Is Confusing, but It Doesn't Have to Be"
+date: 2019-11-25 16:16:51
+tags:
+- CSS
+categories: CSS
+twitter_text: "z-index Is Confusing, but It Doesn't Have to Be"
+authors: Ted Krueger
+image:
```

Comment:
No sweet image?

Reply:
Forgot about that. What says "z-index"?

Reply:
Stack of books? Or some sweet obscure image that makes people think you're some kind of genius?

Reply:
or solitaire?

Reply:
I went with a pile of stacked wood.